src/test_driver/linux-cirque/test-interaction-model.py

```diff
@@ -71,7 +71,7 @@ def run_data_model_test(self):
 
         req_device_id = req_ids[0]
 
-        command = "chip-im-initiator {}"
+        command = "gdb -return-child-result -q -ex run -ex bt chip-im-initiator {}"
```

Comment:
Don't you need `--args chip-im-initiator {}`?

Reply:
Done, thanks